content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -150,40 +156,53 @@ DATABASE_URL=postgresql://test:test@localhost:5432/test?schema=public
 
 When running any command that needs to access the database defined via the `datasource` block (e.g. `prisma introspect`), the Prisma CLI automatically loads the `DATABASE_URL` environment variables from the `.env` file and makes it available to the CLI.
 
-If you want environment variables to be evaluated at runtime, you need to load them manually in your application code, e.g. using [`dotenv`](https://github.com/motdotla/dotenv):
+### Using environment variables in your code
+
+If you want environment variables to be evaluated at runtime, you need to load them manually in your application code (for example, by using [`dotenv`](https://github.com/motdotla/dotenv)):
 
 ```ts
 import * as dotenv from 'dotenv'
 
-dotenv.config() // load the environment variables
+dotenv.config() // Load the environment variables
 console.log(`The connection URL is ${process.env.DATABASE_URL}`)
 ```
 
-### Use multiple .env files or .env files outside the prisma folder
+### Manage `.env` files with `dotenv-cli`
 
-To use multiple `.env` files:
+- Use multiple `.env` files
+- Use `.env` files outside the `prisma` folder
 
-1.  Install [dotenv-cli](https://www.npmjs.com/package/dotenv-cli):
+#### Via command line
+
+1.  Install [`dotenv-cli`](https://www.npmjs.com/package/dotenv-cli):
 
     ```
     npm install -g dotenv-cli
     ```
 
 2.  Create a file - for example, `.env3` - in your project's root folder.
 
-3.  To use the `.env3` file, you can either:
-
-    - Include a reference to `dotenv` at the top of your project's entry file.
+3.  To use the `.env3` file, you can use `dotenv` when you run any Prisma command and specify which `.env` file to use. The following example uses a file named `.env3`:
 
-    ```ts
-    import { config } from 'dotenv'
-    config({ path: '.env3' })
     ```
+    dotenv -e .env3 npx prisma introspect
+    ```
+
+#### Via application code
 
-    - Use `dotenv` when you run any Prisma command and specify which `.env` file to use. The following example uses a file named `.env3`:
+1.  Install [dotenv-cli](https://www.npmjs.com/package/dotenv-cli):
 
     ```
-    dotenv -e .env3 npx prisma introspect
+    npm install -g dotenv-cli
+    ```
```

Comment:
I am not 100% sure this dependency needs to be added to your proejct manually or if it comes already with Prisma Client. @timsuchanek might know.

Reply:
Not in list of dependencies, so I assume not.